src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2FiltersArgumentCollection.java

```diff
@@ -30,25 +30,25 @@
     @Argument(fullName = "max_germline_posterior", optional = true, doc = "Maximum posterior probability that an allele is a germline variant")
     public double maxGermlinePosterior = 0.025;
 
-    @Argument(fullName = "maxAltAllelesThreshold", optional = true, doc="filter variants with too many alt alleles")
+    @Argument(fullName = "maxAltAllelesThreshold", optional = true, doc = "filter variants with too many alt alleles")
```

Comment:
I tried to separate them. But I didn't do it quite right. I already spent too much time on this, I think it's ok for them to be in the same commit. It's been done before in the Mutect code base. I appreciate your enforcing good version control practice though.